src/pages/Reader.tsx

```diff
@@ -24,6 +24,7 @@ interface State {
   chapters: Chapter[],
   page: number,
   pages: Page[],
+  metadata: Metadata | Record<string, never>,
```

Comment:
whats with the `Record<string, never>`?

Reply:
to allow for an empty object

Reply:
the only time itll be empty is before you pull in the metadata file? why don't you just make it nullable

Reply:
How do you do that again

Reply:
metadata: Metadata | null

Reply:
its also fine currently acc - probably safer since its still an object

nm resolving